internal/system/systemstorage.go

```diff
@@ -25,7 +25,7 @@ func InitStore(config config.Config) (Store, error) {
 		err := os.MkdirAll(config.UserKeysDir, 0600)
 
 		if err != nil {
-			return Store{}, errors.New("error, cannot create data store")
+			return SystemStore{}, errors.New("cannot create data store")
```

Comment:
SystemStore does not exist, it is named `Store{}` according to go lint recommendations